setup/setup.ssjs

```diff
@@ -253,7 +253,7 @@
                 { "Name": "date", "FieldType": "Date", "IsRequired": true },
                 { "Name": "timestamp", "FieldType": "Text", "MaxLength": 25, "IsRequired": true, "IsPrimaryKey": true },
                 { "Name": "id", "FieldType": "Text", "MaxLength": 255, "IsRequired": true, "IsPrimaryKey": true },
-                { "Name": "message", "FieldType": "Text", "MaxLength": 4000, "IsRequired": true, "IsPrimaryKey": true },
+                { "Name": "message", "FieldType": "Text", "MaxLength": 4000, "IsRequired": true },
```

Comment:
Another option is to reduce the MaxLength of the message field to be within the PK limits